src/GafferImage/VectorWarp.cpp

```diff
@@ -184,9 +184,9 @@ void VectorWarp::hashEngine( const std::string &channelName, const Imath::V2i &t
 	Warp::hashEngine( channelName, tileOrigin, context, h );
 
 	h.append( tileOrigin );
-	vectorPlug()->dataWindowPlug()->hash( h );
+	h.append( vectorPlug()->dataWindowHash() );
```

Comment:
Looks like this is another one where we could make one GlobalContext if we did that ourselves?